axelor-base/src/main/java/com/axelor/apps/base/service/imports/ImportCityServiceImpl.java

```diff
@@ -278,4 +296,92 @@ private MetaFile extractCityZip(MetaFile dataFile) throws Exception {
 
     return metaFile;
   }
+
+  private StringBuilder extractDataFromDownloadedFile(BufferedReader downloadedCityFileStream)
+      throws IOException {
+
+    HashMap<String, String> regionMap = new HashMap<>();
+    HashMap<String, String> departmentMap = new HashMap<>();
+    HashMap<String, String> cantonMap = new HashMap<>();
+    List<String> cityList = new ArrayList<>();
+
+    String line;
+
+    while ((line = downloadedCityFileStream.readLine()) != null) {
+      String[] values = line.split("\t");
```

Comment:
There are several occurrences of this delimiter. You should create a constant for it.